src/app/business/home/home.component.ts

```diff
@@ -150,6 +199,20 @@ export class HomeComponent implements OnInit {
         });
         this.initBucket2backendAnd2Type();
     }
+    selectOption(){
+        this.formItemCopy = []
+        let selectOptionItem =this.backendForm && this.backendForm.value.type
+        this.formItems.forEach((item)=>{
+            item.arr.forEach((item1)=>{
```

Comment:
Please avoid such variable names as item and item1 in the same method call. This will confuse anyone reading the code.